src/coreclr/nativeaot/Runtime/Full/CMakeLists.txt

```diff
@@ -58,6 +60,7 @@ if (CLR_CMAKE_TARGET_WIN32)
   add_dependencies(standalonegc-enabled.GuardCF aot_eventing_headers)
   add_dependencies(standalonegc-enabled.GuardCF aot_etw_headers)
   add_library(Runtime.ServerGC.GuardCF STATIC ${COMMON_RUNTIME_SOURCES} ${FULL_RUNTIME_SOURCES} ${RUNTIME_SOURCES_ARCH_ASM} ${SERVER_GC_SOURCES} ${RUNTIME_ARCH_ASM_OBJECTS})
+  target_link_libraries(Runtime.ServerGC.GuardCF PUBLIC netintrinsics)
```

Comment:
This may need a separate build of the library, compiled with

Would it be better to just include the .c file in the runtime build instead of plumbing yet another small library through the system?